core/src/test/java/apoc/atomic/AtomicTest.java

```diff
@@ -34,6 +34,44 @@
 		TestUtil.registerProcedure(db, Atomic.class);
 	}
 
+	@Test
+	public void testAddAndSubInteger(){
+		db.executeTransactionally("CREATE (p:Person {name:'Tom'})");
+		try(Transaction tx = db.beginTx()) {
```

Comment:
maybe it's easier to set the age property in the first query directly?

Reply:
I created a new `setProperty` to "force" the data type to be an Integer,
because with `"CREATE (p:Person {name:'Tom', age: 1})"` in `executeTransactionally`,
the `age` property will be under-the-hood turned into `Long`.